src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,45 @@
 import React from 'react';
+import './PostInfo.scss';
+import { Post } from '../types/Post';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    user,
+    comments,
+    title,
+    body,
+  } = post;
+
+  return (
+    <article className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">
+          {title}
+        </h3>
+
+        {user && <UserInfo user={user} />}
+      </div>
+
+      <p className="PostInfo__body">
+        {body}
+      </p>
+
+      {comments.length
+        ? <CommentList comments={comments} />
+        : (
+          <>
+            <hr />
+            <b data-cy="NoCommentsMessage">
+              No comments yet
+            </b>
+          </>
+        )}
```

Comment:
line should be in each post, no only in those which without comments
```suggestion
      </p>
      
      <hr />

      {comments.length
        ? <CommentList comments={comments} />
        : <b data-cy="NoCommentsMessage">
              No comments yet
          </b>
        }
```

Reply:
It seems to me that the line is redundant if we have comments because they stand out against a different background